src/microbe_stage/Microbe.cs

```diff
@@ -662,11 +664,18 @@ public void EmitToxin(Compound agentType = null)
 
         var position = Translation + (direction * ejectionDistance);
 
-        SpawnHelpers.SpawnAgent(props, 10.0f, Constants.EMITTED_AGENT_LIFETIME,
+        SpawnHelpers.SpawnAgent(props, amountEmitted, Constants.EMITTED_AGENT_LIFETIME,
```

Comment:
I think it makes *more* sense also here to pass directly the amount and let the agent code handle conversion from compound amount to how much damage it should do.